pkg/providers/protobuffers/services.go

```diff
@@ -51,10 +57,21 @@ func NewService(descriptor *desc.ServiceDescriptor) *specs.Service {
 		Comment:            descriptor.GetSourceInfo().GetLeadingComments(),
 		Host:               options[HostOption],
 		Transport:          options[TransportOption],
-		Codec:              options[CodecOption],
+		RequestCodec:       options[CodecOption],
```

Comment:
you mean `RequestCodecOption`?

Reply:
I am setting the `CodecOption` by default which could be overridden by the request and response codec options online `:65` and `:70`